### ISSUE TITLE
Alternative / Updated Home Page UI

### ISSUE THREAD

**Issue by salvatoref99** [libre-tube/LibreTube#1850]
### Describe your suggested feature

An updated UI to the home screen would go a long way I believe. Right now there is no featured tab which is found on the likes of Netflix, Optus Sport, Play Store and even ViMusic which is another Github Project that has done its own take on the featured area. I have made a design mockup which I believe would be an awesome Home Page alternative. The only thing I can see being a bit of an issue is that featured video thumbnails will need to have a higher resolution so it doesn't become blurry when scaled, so if that can be fixed I reckon it'll be perfect.  

With this there might need to be a flow on effect throughout the app, so each page will have its own version of the "Featured" text found in the top left, an example of this is in the Subscriptions Page will say "Subscriptions" and the Library Page will say "Library"

![Alternative Home Screen UI](https://user-images.githubusercontent.com/118034225/201854065-623f9978-3f99-45cf-976e-65e59bc9eb5f.png)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
Hey,
I'm a huge fan of your idea of a redesigned home page, thus my take on that with the "steps" I suggest:

1. I would keep the current trending layout available as a tab and just rename it to `Trends` (disabled by default)
2. Then we can create the new `Home` tab similar to your suggestions, however as we don't really have a `Featured` section we would have the following four sections available for the home page:
 - Trending videos
 - Videos from the feed/subscriptions
 - User playlists
 - Bookmarks, depends on #1856 
 ( - Watch history, but I don't think it makes sense to add it there)
3. Clicking on the title, e.g. `What's trending now` will redirect the user to the actual tab, e.g. the `Trends` tab in the example
4. The home page will only show the first twenty items of each category (as that's what we get per API request), maybe we can make it show more when the user scrolled to the end but that won't be a priority for now.

I'm kinda excited of it and as I got some time the next days I'll probably start working on it soon to make it part of the upcoming update which then will be a huge one! :)
Thanks for your efforts with the design drafts, I would be happy if I had any talent with that :stuck_out_tongue_closed_eyes:

**Comment by salvatoref99** [comment]
Awesome Work 🙌 It looks great!!

**Comment by NeeRaj-2401** [comment]
I loved that **swipe-able** "Featured" section in home page and [alternate player ui](https://user-images.githubusercontent.com/118034225/201453867-262338fa-dc43-4e00-b314-3fc8c3a8c7ca.png)

Hence, @Bnyro and @salvatoref99 , as the Current "Whats Trending Now" Section in Home page is having 5 rows and 2 columns,
i think it would be good to make 2 rows and 5 columns **Swipe-able**  
it will make home page **minimal** and also follows the other sections already implemented

**Comment by Bnyro** [comment]
I decided to use a grid layout there to make all those "categories" look differently.
If changed it we had pretty much two similar layouts which would look worse in my opinion, hence I'm a fan of keeping the current layout.

**Comment by Casuallynoted** [comment]
Is there a way to remove the trending now section from the home page entirely?
